src/main/scala/it/pagopa/interop/backendforfrontend/service/types/AttributeRegistryServiceTypes.scala

```diff
@@ -29,6 +31,18 @@ object AttributeRegistryServiceTypes {
     case VERIFIED  => attributeregistrymanagement.client.model.AttributeKind.VERIFIED
   }
 
+  implicit class AttributeKindProcessConverter(private val ak: String) extends AnyVal {
+    def toProcess: AttributeProcess.AttributeKind = ak match {
+      case "CERTIFIED" => AttributeProcess.AttributeKind.CERTIFIED
+      case "DECLARED"  => AttributeProcess.AttributeKind.DECLARED
+      case "VERIFIED"  => AttributeProcess.AttributeKind.VERIFIED
```

Comment:
let's avoid to take strings, because they cannot be checked at compile time.
This function should take a BFF `AttributeKind` instead of a `String`